mapfishapp/src/main/filtered-resources/WEB-INF/jsp/index.jsp

```diff
@@ -4,19 +4,50 @@
 <%@ page import="java.util.*" %>
 <%@ page import="org.georchestra.mapfishapp.ws.Utf8ResourceBundle" %>
 <%@ page import="org.apache.commons.lang.StringUtils" %>
+<%@ page import="org.springframework.web.context.support.WebApplicationContextUtils" %>
+<%@ page import="org.springframework.context.ApplicationContext" %>
+<%@ page import="org.springframework.web.servlet.support.RequestContextUtils" %>
+<%@ page import="org.georchestra.commons.configuration.GeorchestraConfiguration" %>
+
+
 <%@ page contentType="text/html; charset=UTF-8" %>
 <%@ page pageEncoding="UTF-8" %>
 <%@ page isELIgnored="false" %>
 <%
 Boolean anonymous = true;
 Boolean admin = false;
+Boolean georDatadirActivated = false;
+
+String instanceName = null;
```

Comment:
instanceName takes the value from the datadir if available (line 27), or the value specified at compilation level in the shared.maven... (line 50).

Do not forget that the current jsp file is situated in filtered-resources/ which would be copied at compilation time into the right path (src/main/webapp/...), allowing to remain compatible with the "old" system (no datadir).